### PR TITLE
Fixed a crash in member list loading

### PR DESCRIPTION
**Pull request by LorenDB**
It's Hacktoberfest time, and I'm kicking my campaign off with a fix to a crash that could occur if you opened a room separately (but had no room open in the main window) and tried to open its room members.

### CODE REVIEW

File: src/MemberList.cpp [hunk 1]
```diff
@@ -45,7 +45,7 @@ MemberListBackend::addUsers(const std::vector<RoomMember> &members)
     for (const auto &member : members)
         m_memberList.push_back(
           {member,
-           ChatPage::instance()->timelineManager()->rooms()->currentRoom()->avatarUrl(
+           ChatPage::instance()->timelineManager()->rooms()->getRoomById(room_id_)->avatarUrl(
```
Comment (deepbluev7) [inline src/MemberList.cpp:48]: You should probably only do that lookup once before the loop. :3

File: src/MemberList.cpp [hunk 2]
```diff
@@ -42,11 +42,9 @@ MemberListBackend::addUsers(const std::vector<RoomMember> &members)
 {
     beginInsertRows(QModelIndex{}, m_memberList.count(), m_memberList.count() + members.size() - 1);
 
+    auto thisRoom = ChatPage::instance()->timelineManager()->rooms()->getRoomById(room_id_);
```
Comment (deepbluev7) [inline src/MemberList.cpp:45]: I think this could use a null check :D

File: src/MemberList.cpp [hunk 3]
```diff
@@ -43,11 +43,12 @@ MemberListBackend::addUsers(const std::vector<RoomMember> &members)
     beginInsertRows(
       QModelIndex{}, m_memberList.count(), m_memberList.count() + (int)members.size() - 1);
 
-    for (const auto &member : members)
-        m_memberList.push_back(
-          {member,
-           ChatPage::instance()->timelineManager()->rooms()->currentRoom()->avatarUrl(
-             member.user_id)});
+    auto thisRoom = ChatPage::instance()->timelineManager()->rooms()->getRoomById(room_id_);
+    if (thisRoom.isNull())
```
Comment (deepbluev7) [inline src/MemberList.cpp:47]: Well, now the insert is wrong!

Put the condition outside of the begin/endInsertRows and use a return :3

Comment (LorenDB) [reply src/MemberList.cpp:47]: Whoops, I missed a `!` 😬

Comment (deepbluev7) [reply src/MemberList.cpp:47]: No, the bigger issue is that the count of your insert mismatches what you actually insert, which breaks Qt

Comment (LorenDB) [reply src/MemberList.cpp:47]: Well now. That might be a problem. I'll fix shortly.

**Pull request merged**